egs2/aishell4/asr1/local/data.sh

```diff
@@ -0,0 +1,318 @@
+#!/usr/bin/env bash
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+. ./db.sh || exit 1;
+
+ . utils/parse_options.sh || exit 1;
+
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+
+
+#################################################################
+#####             Downloading their git          ################
+#################################################################
+
+
+# Github AISHELL4 : https://github.com/felixfuyihui/AISHELL-4.git
+FOLDER=AISHELL-4   # voir à quel endroit le mettre ! 
+URL=https://github.com/felixfuyihui/AISHELL-4.git
+
+if [ ! -d "$FOLDER" ] ; then
+    git clone "$URL" "$FOLDER"
+    log "git successfully downloaded"
+fi
+
+#pip install -r "$FOLDER"/requirements.txt   # ATTENTION JE CHANGE LEURS REQUIERMENTS POUR METTRE SENTENCE PIECE 0.1.94 AU LIEU DE 0.1.91 QUI FAIT TOUT BUGUER
+# IDEM POUR TORCH JE MET 1.9, voir le vrai git pour les versions d'origine ! 
+# rappel : attention, j'ai modifié leur Git  --> oui, il faut que je le fork
+
+
+#################################################################
+#####            Downloading data and producing lists      ##############
+#################################################################
+
+
+
+if false ; then
+
+    for room_name in "train_L" "train_M" "train_S" 
+    do 
+
+        wget https://www.openslr.org/resources/111/$room_name.tar.gz -P ${AISHELL4}/  
+        
+        
+        tar -xzvf ${AISHELL4}/"$room_name".tar.gz -C ${AISHELL4}/
+        
+
+        # after that untar step, you have one folder "$room_name" with two subfolders : 
+        #   - wav : a list of .flac audio files, each audio file is a conference meeting of about 30 minutes 
+        #   - TextGrid : a list of .TextGrid and .rttm files 
+
+        # then you have to produce a list of the names of the files located in the "$room_name"/wav/ directory 
+        # list should be like : 
+        #/ocean/projects/cis210027p/shared/corpora/AISHELL4/train_L/wav/20200707_L_R001S01C01.flac
```

Comment:
I'm afraid the directory is a bit specific, you may try `/dataset_dir/corpora/aishell4` for other users not in psc.